lib/talents.json

```diff
@@ -371,6 +371,30 @@
             "detail": "1/round, when you hit with a Main Melee weapon, gain 1 Blademaster Die."
           }
         ],
+        "actions": [
+          {
+            "name": "Parry",
+            "activation": "Reaction",
+            "trigger": "You are hit by a melee attack",
+            "detail": "You gain Resistance to all damage, heat, and burn dealt by the attack"
```

Comment:
Minor nitpick: End the sentences with a `.` for consistency with the other actions.